fe/fe-core/src/main/java/com/starrocks/catalog/TableFunction.java

```diff
@@ -49,8 +54,7 @@ public TableFunction(FunctionName fnName, List<String> defaultColumnNames, List<
 
     public static void initBuiltins(FunctionSet functionSet) {
         TableFunction unnestFunction = new TableFunction(new FunctionName("unnest"), Lists.newArrayList("unnest"),
-                Lists.newArrayList(Type.ANY_ARRAY), Lists.newArrayList(Type.ANY_ELEMENT));
-
+                Lists.newArrayList(Type.ANY_ARRAY), Lists.newArrayList(Type.ANY_ELEMENT), true);
```

Comment:
In line 91 of Function.java, there is a todo like `TODO: we don't currently support varargs with no fixed types. i.e. fn(...)`. Can this todo be removed after this pr merged?

Reply:
No, that comment is for scalar function, and scalar function does not currently support multi-type case.